contracts/mocks/MockDAIPermit.sol

```diff
@@ -0,0 +1,128 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+/// dai.sol -- Dai Stablecoin ERC-20 Token
+
+pragma solidity ^0.8.17;
+
+contract MockDAIPermit {
```

Comment:
this is not a mock, it's the actual DAI implementation. Let call it `DAI` in `/tokens/DAI.sol` so we use `/mock` only for real mocks